### PR TITLE
inkscape-extensions.inkstitch: fix editor opening

### PR DESCRIPTION
**Pull request by felbinger**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by felbinger** [comment]
depends on https://github.com/NixOS/nixpkgs/pull/366537, otherwise plugin will not be loaded at all in `inkscape-with-extensions` pkg

**Comment by pluiedev** [comment]
Doesn't seem to fully work: if I click on Extensions > Ink/Stitch > About nothing shows up

**Comment by tropf** [comment]
that's on me, we didn't test properly.
The dependencies have to be present at build time, which we forgot.

The current version (with #366537 cherry-picked) works for me. Does it for you?
Tested: About, Simulator, Params view, PDF export

As I have already maintained a nixpkgs inkstitch out-of-tree for over 2 years, I have added myself to the maintainer list.
(I have no experience with this beyond reading the guide. If this is the wrong place/PR to do that, please tell me.)

File: pkgs/applications/graphics/inkscape/extensions/inkstitch/default.nix [hunk 1]
```diff
@@ -47,6 +30,26 @@ python3.pkgs.buildPythonApplication rec {
...
+in
+python3.pkgs.buildPythonApplication {
+  pname = "inkstitch";
+  inherit version;
+  pyproject = false; # Uses a Makefile (yikes)
+
+  src = fetchFromGitHub {
+    owner = "inkstitch";
+    repo = "inkstitch";
+    rev = "refs/tags/v${version}";
```
Comment (SuperSandro2000) [inline pkgs/applications/graphics/inkscape/extensions/inkstitch/default.nix:43]:
```suggestion
    tags = "v${version}";
```

File: pkgs/applications/graphics/inkscape/extensions/inkstitch/default.nix [hunk 2]
```diff
@@ -51,6 +51,11 @@ python3.pkgs.buildPythonApplication {
 
   inherit dependencies;
 
+  env = {
+    # to overwrite version string
+    GITHUB_REF = "${version}";
```
Comment (SuperSandro2000) [inline pkgs/applications/graphics/inkscape/extensions/inkstitch/default.nix:56]:
```suggestion
    GITHUB_REF = version;
```

File: pkgs/applications/graphics/inkscape/extensions/inkstitch/default.nix (hunk 1 above)
Comment (pluiedev) [reply pkgs/applications/graphics/inkscape/extensions/inkstitch/default.nix:43]: Should be `tag`, singular, no?

Comment (tropf) [reply pkgs/applications/graphics/inkscape/extensions/inkstitch/default.nix]: done in cd87669d6f040611e6e28bfd56c5d5772c40c560

File: pkgs/applications/graphics/inkscape/extensions/inkstitch/default.nix (hunk 2 above)
Comment (tropf) [reply pkgs/applications/graphics/inkscape/extensions/inkstitch/default.nix]: adjusted in e74bd37a8c15f1c96e8f9c8e5728438dafcce4c2

**Review by tropf — APPROVED** [review]

**Pull request merged**